### ISSUE TITLE
config.session_store example should contain :key

### ISSUE THREAD

**Issue by DCrow** [redis-store/redis-rails#69]
Rails 4.1.8.

Setup as follows:
```ruby
MyApplication::Application.config.session_store :redis_store, 
servers: ENV["REDIS_URL"],
expires_in: 30.days
```
Everything was working as it should, except when i close my browser and reopen it, my session is empty. 
Started looking at other session stores, noticed that many of them have `:key` option.
After I setting it up, everything started working properly.

I think `redis-store` should create a default key. Something like 
```ruby
"_#{Rails.application.class.parent_name.downcase}_session"
``` 
or at least write something about it in `Usage`.

**Comment by tubbo** [comment]
I'll definitely update the README to include the key option, and perhaps we need an error when the :key isn't defined. Should probably look at other implementations of the rails cache store to see how they handle a missing :key.

**Comment by DCrow** [comment]
Turns out i was wrong.  
The problem was not the `:key` option. 
It was `:expires_in`. With Rails it doesn't work. It requires `:expire_after`. If it's not setup cookie expire time is set to `Session`. 

Unfortunately  as I added `:key`, I also changed `:expire_after`. Everything was working properly. And so i thought it was the `:key` and discarded other changes. Then my test team returned it to me saying it didn't work. And here we are...

As for `:key` option, looking through chrome console, i found that Rails defaults to [`_session_id`](https://github.com/rails/rails/blob/4-1-stable/actionpack/lib/action_dispatch/middleware/session/abstract_store.rb#L23)
